.editorconfig

```diff
@@ -0,0 +1,17 @@
+# Drupal editor configuration normalization
```

Comment:
The task runner is not Drupal-specific so it is not correct to mention that here.